todo/src/components/Form.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {addTask} from '../actions';
+import './todo.css'
+
+class Form extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            todo: ''  
+    }}
+
+    handleInputChange = e => {
+        this.setState({todo: e.target.value})
+      };
+
+      addTodo = e => {
+          e.preventDefault();
+          this.props.addTask(this.state.todo)
+          this.setState({todo: ''})
+          document.querySelector('#input').value = ''
+      }
+
+    render() {
+        return (
+        <div className="form-container">
+            <form onSubmit={this.addTodo} autoComplete="off" className="todo-form">
+            <input
+                onChange={this.handleInputChange}
+                placeholder="Task"
+                value={this.state.name}
+                name="name"
+                type="text"
+                id="input"
```

Comment:
I highly recommend keeping your `value` and `name` attributes consistent with the variables you have declared in `state` (and you also shouldn't need the `id` attribute here). Use `"todo"` instead of `"name"` (which is not defined in `state`).